Tribler/Core/Utilities/utilities.py

```diff
@@ -487,4 +487,4 @@ def translate_peers_into_health(peer_info_dicts):
 
 def unichar_string(text):
     """ Unicode character interpretation of text for Python 2.7 """
-    return u''.join(unichr(ord(t)) for t in text)
+    return u''.join(six.unichr(ord(t)) for t in text)
```

Comment:
We can replace this method entirely with this: https://github.com/Tribler/tribler/pull/4094#discussion_r240111555

Reply:
I am unclear what you mean.

Reply:
Well I guess it's not really relevant to your changes. But this function does exactly what I linked in that comment. At some point (probably not in this PR), we should replace our custom stuff with builtin library functionality.